apiserver/facades/client/modelconfig/modelconfig.go

```diff
@@ -269,7 +281,29 @@ func (c *ModelConfigAPI) SetModelConstraints(ctx context.Context, args params.Se
 	if err := c.check.ChangeAllowed(ctx); err != nil {
 		return errors.Trace(err)
 	}
-	return c.backend.SetModelConstraints(args.Constraints)
+	err := c.modelSericve.SetModelConstraints(ctx, args.Constraints)
+	if errors.Is(err, modelerrors.NotFound) {
+		return apiservererrors.ParamsErrorf(
+			params.CodeModelNotFound,
+			"model %q not found",
+			c.modelUUID,
+		)
+	}
+	if errors.Is(err, networkerrors.SpaceNotFound) {
```

Comment:
We should include the value that is the problem and potentially ditch the model id ?

Reply:
What I can probably do is to include the original error message.